### ISSUE TITLE
Missing TH on admin form tables

### ISSUE THREAD

**Issue by rachidatecs** [cisagov/manage.get.gov#2941]
### Current Behavior

The custom tables we're using throughout DJA for contacts feature a 'copy email' column on the right. That column 's TDs do not have any associated TH. 

### Expected Behavior

Every TD should have an associated TH either in the THEAD or as the first table cell in its row. 

Per [this slack discussion](https://cisa-corp.slack.com/archives/C05BDEA3C11/p1728419979311949), we should have a  visible TH that says ‘Action’

This should be applied to every instance of DJA table that features a 'copy email' button.

### Steps to Reproduce

1. Navigate to a domain request with other contacts or portfolio with members and admins.
2. Inspect table.

**Comment by abroddrick** [comment]
Soft sprinting for 57, just because it's a nice easy ticket that shouldn't take a ton of time

**Comment by SamiyahKey** [comment]
I have completed the Design Review for this issue [3071](rjm/2941-dja-table-headers), it looks great and passes all acceptance criteria!
